src/darwin/Framework/CHIP/MTRDeviceController.h

```diff
@@ -119,6 +120,14 @@ typedef void (^MTRDeviceConnectionCallback)(MTRBaseDevice * _Nullable device, NS
  */
 - (void)setPairingDelegate:(id<MTRDevicePairingDelegate>)delegate queue:(dispatch_queue_t)queue;
 
+- (void)setNocChainIssuer:(id<NOCChainIssuer>)nocChainIssuer;
+
+- (uint32_t)onNOCChainGeneration:(NSData *)operationalCertificate
```

Comment:
Needs to be NSNumber *, and a better name.

Reply:
Done and method name updated